components/SponsorsSection.js

```diff
@@ -24,6 +24,7 @@ export default function TicketsSection () {
           {sponsorsGold.map(name => (
             <ItemSponsor
               key={name}
+              name={name}
```

Comment:
The name here is the file name, and is not a correct name to refer to the sponsor.

Here are the sponsor names:

- Thai Programmer Association
- SPRINT3r
- Cleverse
- int’genxe
- Omise
- Pronto Tools
- Taskworld
- THiNKNET
- Wongnai

I suggest changing the array, instead of a string (`taskworld`), use an object (`{ name: 'Taskworld', image: 'taskworld' }`).